_posts/en/newsletters/2022-11-16-newsletter.md

```diff
@@ -0,0 +1,189 @@
+
+- **Blixt Wallet adds taproot support:**
+  [Blixt Wallet v0.6.0][blixt v0.6.0] adds send and receive support for [taproot][topic taproot] addresses.
+
+- **Specter-DIY v1.8.0 released:**
+  [Specter-DIY v1.8.0][] now supports [reproducible builds][topic reproducible
+  builds] and [taproot][topic taproot] keypath spending support.
+
+- **Trezor Suite adds coin control features:**
+  In a [recent blog post][trezor coin control], Trezor announced that Trezor
+  Suite now supports [coin control][topic coin selection] features.
+
+- **Strike adds taproot send support:**
+  Strike's wallet now allows sending to [bech32m][topic bech32] addresses.
+
+- **Kollider exchange launches with Lightning support:**
+  Kollider [announced][kollider launch] an exchange with LN deposit and
+  withdrawal capabilities as well as a browser-based Lightning wallet.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC4][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
+
+  **Warning:** this release candidate includes the `mempoolfullrbf`
+  configuration option which several protocol and application developers
+  believe could lead to problems for merchant services as described in
+  previous newsletters [#222][news222 rbf] and [#223][news223 rbf].  It
+  could also cause problems for transaction relay as described in
+  [Newsletter #224][news224 rbf].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Core Lightning #5681][] adds the ability to filter the JSON output of
+  an RPC on the server side.  Filtering on the server side avoids
+  sending unwanted data when using a bandwidth constrained remote
+  connection.  In the future, some RPCs may be able to avoid computing
+  filtered data, allowing them to return sooner.  Filtering is not
+  guaranteed for all RPCs, especially those provided by plugins.  When
+  filtering is not available, the unfiltered full output will be
+  returned.  For more information, see the added [documentation][cln
+  filter doc].
+
+- [Core Lightning #5698][] updates the experimental developer mode to
+  allow receiving onion-wrapped error messages of any size. BOLT2
+  currently recommends 256 byte errors, but doesn’t forbid longer error
+  messages and [BOLTs #1021][] is open to encourage use of 1024-byte
```

Comment:
```suggestion
  messages, and [BOLTs #1021][] is open to encourage use of 1024-byte
```

Reply:
Edited previous line so this one can stay the same.